tests/pass-dep/libc/libc-time.rs

```diff
@@ -1,5 +1,6 @@
 //@ignore-target-windows: no libc time APIs on Windows
 //@compile-flags: -Zmiri-disable-isolation
+#[cfg(any(target_os = "linux", target_os = "macos", target_os = "freebsd"))]
```

Comment:
Instead of making the import conditional, please remove it entirely and use the fully expanded type name below where `CStr` shows up.